backend/services/client.js

```diff
@@ -43,7 +43,7 @@ exports.loginService = async function ({ email, password, type }) {
     })
   ).toObject();
 
-  return Formatters.formatClientToken({ ...newClientToken, client: clientWithEmail });
+  return Formatters.formatClientToken({ ...newClientToken, client });
```

Comment:
```suggestion
  return Formatters.formatClientToken({ ...newClientToken, client: clientWithEmail  });
```
This was correct, I will revert this change